python-regression/util/static_vals.py

```diff
@@ -121,3 +122,49 @@
+    "NHFBHTUJHNXDIKYCGGKDVJNYP999NELADUXCEXLKFRWGVCIVVLYPPFZUFXAZURTXJK9SUW9OCWHGAGMZ9"
+]
+
+NULL_TRANSACTION = "999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+99999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999999\
+999999999999999999999999999999999999999999999999999999999999999999999999999"
+
+NULL_TRANSACTION_LIST = [TryteString(NULL_TRANSACTION), TryteString(NULL_TRANSACTION), TryteString(NULL_TRANSACTION)]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['TryteString' may be undefined, or defined from star imports: iota](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=3727297)